pkgs/development/python-modules/snapcast/default.nix

```diff
@@ -1,24 +1,32 @@
 { lib
 , buildPythonPackage
 , construct
-, fetchPypi
+, fetchFromGitHub
 , isPy3k
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "snapcast";
-  version = "2.1.2";
+  version = "2.1.3";
   disabled = !isPy3k;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-ILBleqxEO7wTxAw/fvDW+4O4H4XWV5m5WWtaNeRBr4g=";
+  src = fetchFromGitHub {
+    owner = "happyleavesaoc";
+    repo = "python-snapcast";
+    # The current release has a wrong tag, missing period 2.13 instead 2.1.3
```

Comment:
https://github.com/happyleavesaoc/python-snapcast/issues/39

Reply:
Thanks for opening the issue.

Reply:
Fixed upstream, please update.